sdk/python/feast/feature_store.py

```diff
@@ -1078,15 +1079,15 @@ def get_online_features(
 
         return self._get_online_features(
             features=features,
-            entity_values=columnar,
+            entity_values=dict(columnar),
             full_feature_names=full_feature_names,
             native_entity_values=True,
         )
 
     def _get_online_features(
         self,
         features: Union[List[str], FeatureService],
-        entity_values: Dict[str, List[Any]],
+        entity_values: Dict[str, Union[Sequence[Any], Sequence[Value], RepeatedValue]],
```

Comment:
any reason why you have Sequence[Value] sinec you already have Any?

Reply:
It was deliberate as I thought it was more communicative of what is actually expected. My interpretation of our use of `Any` in many places is 'any native Python type' rather than actually any class. So in my mind `Sequence[Any]` is different to `Sequence[Value]`. And then `RepeatedValue` is another separate case. 

Flexible on that though.

Reply:
Fair enough. Mind updating the docstrings for the get_online_features method then to reflect this?

Reply:
`get_online_features` hasn't really changed at all. I believe we'd still expect the user to pass native Python types. However, `get_online_features` now wraps `_get_online_features` which does expect to have to be able to handle Protobuf types. 

Do you still think the Docstring needs updating? Nothing has changed for the user.

Reply:
ah nvm. I had quickly skimmed and thought you also allowed users to technically pass values in the regular API. LG!